docs/source/index.rst

```diff
@@ -48,6 +48,7 @@ The :mod:`torchaudio` package consists of I/O, popular datasets and common audio
 
    auto_examples/wav2vec2/index
    auto_examples/tts/index
+   auto_examples/mvdr/index
```

Comment:
I think that the structure of examples can be improved, let me follow up on that later.